src/asset_folder_vsingester.py

```diff
@@ -39,8 +40,25 @@ def file_has_any_extension(filename, extensionlist):
 
     return False
 
+
+def test_portal_connection(filepath, cfg):
```

Comment:
a little docstring here would be nice, explaining that the function returns normally if a connection could be established and raises SweeperHTTPError if no connection could be established

Reply:
Done.